docs/docs/building/rollup-plugin-html.md

```diff
@@ -105,7 +105,7 @@ export default {
 
 ### Bundling assets
 
-The HTML plugin will bundle assets referenced from `img` and `link` elements in your HTML. The assets are emitted as rollup assets, and the paths are updated to the rollup output paths.
+The HTML plugin will bundle assets referenced from `img` and `link` and `meta property="og:image"` elements in your HTML. The assets are emitted as rollup assets, and the paths are updated to the rollup output paths.
```

Comment:
```suggestion
The HTML plugin will bundle assets referenced from `img` and `link` and social media tags in your HTML. The assets are emitted as rollup assets, and the paths are updated to the rollup output paths.
```

The code examples shows it handles more than just `og:image`

Reply:
changed